templater/jobs/postsubmit/eks-distro-build-tooling/prow-plugin-postsubmit.yaml

```diff
@@ -11,3 +12,10 @@ resources:
   requests:
     cpu: 2560m
     memory: 16Gi
+envVars:
+- name: IMAGE_REPO
+  value: public.ecr.aws/eks-distro-build-tooling
```

Comment:
I think we want this to be `public.ecr.aws/h1r8a7l5`

Reply:
Oh yeah because we should make this one the internal public and add a PROD job for this repo. Then in that case I shouldn't need the source call to the script correct?